multiqc/utils/search_patterns.yaml

```diff
@@ -13,6 +13,10 @@ afterqc:
 anglerfish:
   fn: "*.json"
   contents: "anglerfish_version"
+bakta:
+  fn: "*.txt"
+  contents: "Bakta:"
+  num_lines: 26
```

Comment:
```suggestion
```
Is there a chance that in some circumstances or in future Bakta versions there will be more metrics reported, and the "Bakta:" line will be pushed below the line 26? Perhaps makes sense to read more lines to be safe. Let's maybe remove it and just make it read the default 1000 lines